direct/nn/mri_models.py

```diff
@@ -141,7 +141,12 @@ def l1_loss(source: torch.Tensor, reduction: str = "mean", **data) -> torch.Tens
                 L1 loss.
             """
             resolution = get_resolution(**data)
-            l1_loss = F.l1_loss(*_crop_volume(source, data["target"], resolution), reduction=reduction)
+            l1_loss = F.l1_loss(
+                *_crop_volume(
+                    T.modulus_if_complex(source, complex_axis=self._complex_dim), data["target"], resolution
+                ),
+                reduction=reduction,
+            )
```

Comment:
Have always been thinking about being a bit more flexible here. Perhaps we can look into making it configurable and allow for complex loss?

Reply:
Yes sure. But on a new issue/PR